docs/integrations.md

```diff
@@ -70,6 +70,21 @@ jobs:
 This will default to using the [`r2c` rule pack](https://semgrep.live/p/r2c).
```

Comment:
I don’t think you need `r2c` here styled this way, otherwise we’ll have to style all of them like this in these instructions, in the card views, etc.